WebApplication/ClientApp/src/components/forgePdfView.js

```diff
@@ -72,15 +72,43 @@ export class ForgePdfView extends Component {
         if (!this.props.drawingPdf)
             return;
 
-        this.viewer.loadModel( this.props.drawingPdf, { page: 1 }); // load page 1 by default
+        this.viewer.loadModel( this.props.drawingPdf, { page: 1 } ); // load page 1 by default
         //this.viewer.loadExtension("Autodesk.Viewing.MarkupsCore")
         //this.viewer.loadExtension("Autodesk.Viewing.MarkupsGui")
     }
 
     componentDidUpdate(prevProps) {
         if (this.viewer && Autodesk && (this.props.drawingPdf !== prevProps.drawingPdf)) {
-            this.viewer.loadModel( this.props.drawingPdf, { page: 1 }); // load page 1 by default
-            this.viewer.unloadModel(prevProps.drawingPdf);
+
+            if (prevProps.drawingPdf != null) {
+                // try to find model in allModels
+                const allModels = this.viewer.getAllModels();
+                allModels.forEach(model => {
```

Comment:
It's NBD, but `for` loop could be more suitable here, because it's possible to `break` loop if required model is found.
// same below